src/maintainer/knowledge_base.rst

```diff
@@ -203,7 +231,7 @@ To skip building with a particular ``vc`` version, add a skip statement.
         - {{ compiler('cxx') }}
 
 Using vs2019
--------------
+^^^^^^^^^^^^^
```

Comment:
Since you're touching this section, I'll note that it's outdated and should be updated; https://github.com/conda-forge/conda-forge.github.io/pull/1878 has been open for a while, I guess it could be folded into this PR?

Reply:
My best shot at that: e99d98c2